### PR TITLE
Pay-2939/2940/2941 : ITHC issues fixed for Payment API

### PR DESCRIPTION
**Pull request by AyanDuttaHMCTS**
### JIRA link (if applicable) ###

Pay-2939/2940/2941 : ITHC issues fixed for Payment API

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: api/src/test/java/uk/gov/hmcts/payment/api/unit/SendMessageApplication.java [hunk 1]
```diff
@@ -37,6 +37,7 @@ public static void main(String[] args) throws Exception{
         msg.setProperties(Collections.singletonMap("serviceCallbackUrl", payment.getServiceCallbackUrl()));
 
         client.send(msg);
+        /*client.send(msg);
```
Comment (YasharRahvar) [inline api/src/test/java/uk/gov/hmcts/payment/api/unit/SendMessageApplication.java:40]: remove the unnecessary comments

File: api/src/test/java/uk/gov/hmcts/payment/api/componenttests/jobs/MaintenanceJobsControllerTest.java [hunk 2]
```diff
@@ -127,7 +127,7 @@ public void testThatServiceCallbackIsInvokedWhenAStatusChangeIsDetectedOnGovPay(
         // Create Payment
 
         MvcResult result = restActions
-            .withReturnUrl("https://www.google.com")
+            //.withReturnUrl("https://www.moneyclaims.service.gov.uk")
```
Comment (YasharRahvar) [inline api/src/test/java/uk/gov/hmcts/payment/api/componenttests/jobs/MaintenanceJobsControllerTest.java:130]: remove the unnecessary comments

File: api/src/test/java/uk/gov/hmcts/payment/api/componenttests/CaseControllerTest.java [hunk 3]
```diff
@@ -431,7 +436,7 @@ public void getAllPaymentGroupsHavingMultipleFeesRemissionsAndPaymentsWithCcdCas
             .build();
 
         MvcResult result1 = restActions
-            .withReturnUrl("https://www.google.com")
+            //.withReturnUrl("https://www.moneyclaims.service.gov.uk")
```
Comment (YasharRahvar) [inline api/src/test/java/uk/gov/hmcts/payment/api/componenttests/CaseControllerTest.java:439]: remove the unnecessary comments

File: api/src/test/java/uk/gov/hmcts/payment/api/componenttests/CaseControllerTest.java [hunk 4]
```diff
@@ -536,7 +541,7 @@ public void validateNewlyAddedFieldsInPaymentGroupResponse() throws Exception {
...
```
Comment (YasharRahvar) [inline api/src/test/java/uk/gov/hmcts/payment/api/componenttests/CaseControllerTest.java:544]: remove the unnecessary comments

Comment (AyanDuttaHMCTS) [reply api/src/test/java/uk/gov/hmcts/payment/api/componenttests/CaseControllerTest.java]: Clean up Done, Thanks for Highlighting.

File: api/src/test/java/uk/gov/hmcts/payment/api/componenttests/CaseControllerTest.java (hunk 3 above)
Comment (AyanDuttaHMCTS) [reply api/src/test/java/uk/gov/hmcts/payment/api/componenttests/CaseControllerTest.java]: Clean up Done, Thanks for Highlighting.

File: api/src/test/java/uk/gov/hmcts/payment/api/componenttests/jobs/MaintenanceJobsControllerTest.java (hunk 2 above)
Comment (AyanDuttaHMCTS) [reply api/src/test/java/uk/gov/hmcts/payment/api/componenttests/jobs/MaintenanceJobsControllerTest.java]: Clean up Done, Thanks for Highlighting.

File: api/src/test/java/uk/gov/hmcts/payment/api/unit/SendMessageApplication.java (hunk 1 above)
Comment (AyanDuttaHMCTS) [reply api/src/test/java/uk/gov/hmcts/payment/api/unit/SendMessageApplication.java]: Clean up Done, Thanks for Highlighting.

**Review by YasharRahvar — APPROVED** [review]

**Review by krishnanuthalapati — APPROVED** [review]

**Pull request merged**